index.js

```diff
@@ -121,7 +121,10 @@ class DockerExecutor extends Executor {
      * @return {Promise}
      */
     _start(config) {
-        const [buildImage, buildTag] = config.container.split(':');
+        const containerName = config.container.split(':');
+
+        const buildTag = (containerName.length === 1) ? 'latest' : containerName.pop();
+        const buildImage = containerName.join(':');
```

Comment:
Included the `docker-parse-image` module to do most of the image name parsing. 

A side-effect is that the default images are prefixed with `library/` as the namespace. Since that was implied before, the `docker-parse-image` module makes it explicit. So the other tests had to be modified to account for this change.